charts/prometheus-operator-crds/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 type: application
-version: 0.1.1
+version: 0.1.2
```

Comment:
not sure if a bugfix version would be enough, if the crd changes.
imho it would be even a major version update.

Reply:
Yes, I understand that. I am following the previous update that increased the patch number whilst increasing appVersion as well. I am hoping the code owner will express their opinion on leaving major number 0 and bumping it up.

Reply:
I reckon we could increase at least the minor number at this moment.

Reply:
We could follow the same increases than appVersion but one layer upper:
- 0.50.0 -> 0.50.1 would increase a minor version
- 0.50.0 -> 0.51.0 would increase a major version (since it's often breaking)
- 0.50.0 -> 0.50.0 but with internal, non breaking, non features changes would increase patch version
What do you think?

Reply:
@desaintmartin Thank you. In the current update to 0.62.0, we would thus have 1.0.0. This would be inline with the suggestion by @monotek on a major number bump.